rules/linux/persistence_shared_object_creation.toml

```
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2023/06/09"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup, New Term"
+min_stack_version = "8.6.0"
+updated_date = "2023/06/09"
+
+[rule]
+author = ["Elastic"]
+description = """
+The creation of a shared object file involves compiling code into a dynamically linked library that can be loaded by 
+other programs at runtime. While this process is typically used for legitimate purposes, malicious actors can leverage
+shared object files (e.g., .so files) to execute unauthorized code, inject malicious functionality into legitimate
+processes, or bypass security controls. This allows malware to persist on the system, evade detection, and potentially
+compromise the integrity and confidentiality of the affected system and its data. This rule monitors the creation of
+shared object files by previously unknown processes. 
```

```suggestion
This rule monitors the creation ofshared object files by previously unknown processes. 
The creation of a shared object file involves compiling code into a dynamically linked library that can be loaded by 
other programs at runtime. While this process is typically used for legitimate purposes, malicious actors can leverage
shared object files (e.g., .so files) to execute unauthorized code, inject malicious functionality into legitimate
processes, or bypass security controls. This allows malware to persist on the system, evade detection, and potentially
compromise the integrity and confidentiality of the affected system and its data. 
```
maybe consider moving your last line to the front.